platforms/documentation/docs/src/docs/userguide/dep-man/getting_started_dep_man.adoc

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_in_gradle]]
+= Dependency Management
+
+Software projects typically rely on libraries, internal modules and/or external modules to function.
```

Comment:
```suggestion
Software projects typically depend on other libraries to function. 
These libraries can either be sourced from other projects in the same build or from external repositories.
```

What is the distinction here between a library and a module? 

I think we should be careful with what we mean by "module" all throughout the documentation.